subsys/bluetooth/audio/otc_internal.h

```diff
@@ -0,0 +1,36 @@
+/* @file
+ * @brief Object Transfer client internal header file
+ *
+ * For use with the Object Transfer Service Client (OTC)
+ *
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
```

Comment:
Given that this file only implements a single macro (which I suppose may as well be defined in the public otc.h or even ots.h files) that is only used a single place, maybe this file is overkill to have.